### ISSUE TITLE
This module does not build

### ISSUE THREAD

**Issue by rayjlinden** [caddy-dns/route53#10]
First saw the issue in xcaddy.

However, even if I try to just "get" the module I get this error:

```
go get github.com/caddy-dns/route53
go: downloading github.com/caddy-dns/route53 v1.1.0
go: github.com/caddy-dns/route53 upgrade => v1.1.0
go: downloading github.com/libdns/route53 v1.0.1
# github.com/caddy-dns/route53
../../../../../../pkg/mod/github.com/caddy-dns/route53@v1.1.0/route53.go:32:12: p.Provider.AWSProfile undefined (type *"github.com/libdns/route53".Provider has no field or method AWSProfile)
../../../../../../pkg/mod/github.com/caddy-dns/route53@v1.1.0/route53.go:60:16: p.Provider.AWSProfile undefined (type *"github.com/libdns/route53".Provider has no field or method AWSProfile)
```

**Comment by rayjlinden** [comment]
I think someone forgot to checkin the go.mod file.

It is referencing github.com/libdns/route53 v1.0.1
(Which does not not have AWSProfile as a member of the struct.)

But clearly the new code wants to use it and it is available in version 1.1 of the module.

**Comment by rayjlinden** [comment]
Here is a PR with the fix: https://github.com/caddy-dns/route53/pull/11
